pkg/dockerregistry/server/audit/log.go

```diff
@@ -0,0 +1,101 @@
+package audit
+
+import (
+	"sync"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/distribution/context"
+)
+
+const (
+	LogEntryType     = "openshift.log.entry"
```

Comment:
`openshift.logger` can be on of `{ default | audit }`

Instead of `default`, we may choose `base` or `generic`. Any better ideas?